CIP-0XXX/README.md

```diff
@@ -0,0 +1,551 @@
+| 5 min       | 12 kB/s     | 28 kB/s     |
+| 10 min      | 6 kB/s      | 14 kB/s     |
+
+- the network outbound volume of a peer is:
+
+| Send period | Lower bound  | Upper bound  |
+| ----------- | ------------ | ------------ |
+| 1 min       | 147 GB/month | 356 GB/month |
+| 2 min       | 74 GB/month  | 178 GB/month |
+| 5 min       | 30 GB/month  | 72 GB/month  |
+| 10 min      | 15 GB/month  | 36 GB/month  |
+
+### Infrastructure extra operating costs
+
+#### Networking traffic cost
+
+> [!NOTE]  
+> - These data apply to cloud providers which bill the traffic on the volume, not the bandwidth.
+> - Some cloud providers offer a free tier for the first **100GB** of traffic which is not taken into consideration here for simplicity.
+
+| Cloud Provider | Inbound Traffic | Outbound Traffic |
+|--------|------|------|
+| AWS | 0 $/GB | 0.09 $/GB |
+| GCP | 0 $/GB | 0.11 $/GB |
+| Azure | 0 $/GB | 0.09 $/GB |
+| Average | 0 $/GB | 0.1 $/GB |
+
+
+#### Mithril message diffusion extra networking cost
+
+For a total of `3,000` SPOs sending messages, the extra networking cost incurred for a Cardano full node is:
+
+| Send period | Lower bound | Upper bound |
+| ----------- | ----------- | ----------- |
+| 1 min       | 15 $/month  | 36 $/month  |
+| 2 min       | 8 $/month   | 18 $/month  |
+| 5 min       | 3 $/month   | 8 $/month   |
+| 10 min      | 2 $/month   | 4 $/month   |
+
+### Possible attacks
+
+#### Sybil attack
+
+In this attack, a malicious sender would attempt to create multiple identities impersonating SPOs. This attack is completely mitigated by the [Message authentication mechanism](#Message-authentication-mechanism) as only active SPO on the Cardano chain can be authenticated and send messages. This would be considered as a protocol violation and the malicous peer would be disconnected.
+
+#### Equivocation
+
+In this attack, a malicious SPO would send different messages to different peers. This attack needs to be handled by the receiver of the message as the network layer does not verify the content of the message body by design.
+
+In the specific case of Mithril, the individual signature is unique so there will be two cases:
+- the message embeds a valid signature and it will be accepted by the receiving Mithril aggregator.
+- the message embeds an invalid signature and it will be rejected by the receiving Mithril aggregator.
+
+#### DoS attack
+
+In this attack, a malicous SPO would try to flood the network by sending many messages at once. In that case, the network layer could detect that the throughput of messages originating from a SPO is above a threshold and consider it as a protocol violation, thus disconnecting the malicous peer.
+
+## Local Message Submission mini-protocol
+
+### Description
+
+The local message submission mini-protocol is used by local clients to submit message to a local node. This mini-protocol is **not** used to diffuse messages from a core node to another. 
```

Comment:
> from a core node to another

I don't like this wording as it hints that the protocol will run on cardano-node.

Reply:
From our discussions, the cardano node would be called directly by the n2c mini-protocols

Reply:
yes but it won't be cardano-node running the protocols right? You'll just be querying its chain/state